src/lib/operators/join_nested_loop.cpp

```diff
@@ -154,7 +155,7 @@ void JoinNestedLoop::_perform_join() {
 
           // clang-format off
           if constexpr (neither_is_string_column || both_are_string_columns) {
-            auto iterable_left = create_iterable_from_column<LeftType>(typed_left_column);
+            auto iterable_left = erase_type_from_iterable(create_iterable_from_column<LeftType>(typed_left_column));
```

Comment:
The idea is here to always use an `AnyColumnIterable` for the left column because the nested-loop join is only used for very few elements and it doesn’t make a huge difference if it’s a bit slower if it compiles faster. I can of course remove it.